astroplan/tests/test_target.py

```diff
@@ -40,3 +43,38 @@ def test_FixedTarget_ra_dec():
                                                         'SkyCoord')
     assert vega.coord.dec == vega_coords.dec == vega.dec, ('Retrieve Dec from '
                                                            'SkyCoord')
+
+
+def test_get_skycoord():
+    m31 = SkyCoord(10.6847083*u.deg, 41.26875*u.deg)
+    m31_with_distance = SkyCoord(10.6847083*u.deg, 41.26875*u.deg, 780*u.kpc)
+    subaru = Observer.at_site('subaru')
+    time = Time("2016-01-22 12:00")
+    pos, vel = subaru.location.get_gcrs_posvel(time)
+    gcrs_frame = GCRS(obstime=Time("2016-01-22 12:00"), obsgeoloc=pos, obsgeovel=vel)
+    m31_gcrs = m31.transform_to(gcrs_frame)
+    m31_gcrs_with_distance = m31_with_distance.transform_to(gcrs_frame)
+
+    coo = get_skycoord(m31)
+    assert coo.is_equivalent_frame(ICRS())
+    assert_raises(TypeError, len, coo)
```

Comment:
Could you please use pytest.raises instead? We don't have nose as a mandatory requirement but pytest, and this currently fails with ``ImportError: Need nose >= 1.0.0 for tests - see http://somethingaboutorange.com/mrl/projects/nose``